components/Templates/includes/auto-template/Pods_Templates_Auto_Template_Front_End.php

```diff
@@ -75,7 +75,7 @@ public function hook_content() {
 			// No need to check for default hooks, this is already done in auto_pods().
 			$filter = pods_v( 'single_filter', $possible_pods[ $pod ], '', true );
 			if ( $filter ) {
-				$this->filtered_content[ $filter ] = 10.5;
+				$this->filtered_content[ $filter ] = 10;
```

Comment:
I wonder if we should set this to 11 or not

Reply:
I choose 10 because when I tested `(int) 10.5` it returned 10. But yeah, I had the same thought myself haha.
However, we allow users to override this value so personally I wouldn't worry about it too much and leave it as the WP default value.